batch/batch/database.py

```diff
@@ -13,9 +13,10 @@ def run_synchronous(coro):
 
 @asyncinit
 class Database:
-    @staticmethod
-    def create_synchronous(config_file):
-        db = run_synchronous(Database(config_file))
+    @classmethod
+    def create_synchronous(cls, config_file):
+        db = object.__new__(cls)
+        run_synchronous(cls.__init__(db, config_file))
```

Comment:
can you comment on why this is necessary?

Reply:
I think this is clearer than `run_synchronous(Database(config_file))`.